docs/design/uplink-scopes.md

```diff
@@ -18,7 +18,7 @@ Uplink will operate on a set of "scopes". Each scope contains an API key, a sate
 type Scope struct {
     APIKey        string // Or perhaps uplink.APIKey
     SatelliteURL  string
-    ProjectKey    *string
+    ProjectKey    []byte
```

Comment:
"RootEncryptionKey", maybe? I have a tendency to get confused about unqualified "keys" in our codebase.

Reply:
"ProjectEncryptionKey"?